tex/generic/pgf/graphdrawing/lua/pgf/gd/control/library.lua

```diff
@@ -53,11 +53,16 @@ declare {
     This behavior can be changed using this option. When the key is
     invoked, nodes are placed \emph{behind} the edges.
   "]],
-  examples = [["
-    \tikz \graph [simple necklace layout, nodes={draw,fill=white},
-                  nodes behind edges]
-      { subgraph K_n [n=7], 1 [regardless at={(0,-1)}] };
-  "]]
+  examples = {
+    {
+      options = [["preamble={\usetikzlibrary{graphs.standard,graphdrawing} \usegdlibrary{circular}}"]],
```

Comment:
I'm not sure whether I merged the support for this new syntax yet. I have to check that before merging.

Reply:
Yes, you had. At least it seems to work in the currently available first example of

https://github.com/pgf-tikz/pgf/blob/0280303d03594e1ea13f95f116234a803591b4af/tex/generic/pgf/graphdrawing/lua/pgf/gd/force/ControlElectric.lua#L42-L60

which shows as

https://pgf-tikz.github.io/pgf/pgfmanual.pdf#page=477